.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,4 +5,5 @@
 - [ ] Run `bundle exec rubocop` to test for code style violations and recommendations
 - [ ] Add Unit Tests (aka `specs/*_spec.rb`) if applicable
 - [ ] Run `bundle exec rspec` to run the whole test suite and ensure all your tests pass
-- [ ] Make sure you added an entry in [the `CHANGELOG.md` file](https://github.com/wordpress-mobile/release-toolkit/blob/trunk/CHANGELOG.md#trunk) to describe your changes under the approprioate existing `###` subsection of the existing `## Trunk` section.
\ No newline at end of file
+- [ ] Make sure you added an entry in [the `CHANGELOG.md` file](https://github.com/wordpress-mobile/release-toolkit/blob/trunk/CHANGELOG.md#trunk) to describe your changes under the appropriate existing `###` subsection of the existing `## Trunk` section.
```

Comment:
@AliSoftware: I noticed a small typo in the `.github/PULL_REQUEST_TEMPLATE.md` ("approprioate"), and while editing it I realised it would be nice to mention the `MIGRATION.md` here as well. Please let me know if you think this makes sense, otherwise I can just revert it.